### ISSUE TITLE
sonar-audit should check for branches that have been analyzed only once

### ISSUE THREAD

**Issue by okorach** [okorach/sonar-tools#959]
In the life of a project, each long living branch code is supposed to evolve over time.
If the branch has been analyzed only once in a given period - 6 months by default - and the branch is marked as keep when inactive) i's likely to be an accidental analysis of a branch that should not be kept. sonar-audit should issue a warning

cc @sylvain-combe-sonarsource

**Comment by okorach** [comment]
This is already verified by branches whose last analysis is too old, so I am closing this as won't fix
